src/main/java/data_access/WeekDataAccess.java

```diff
@@ -41,14 +42,58 @@ public WeekDataAccess(ArrayList<Day> dataDays){
         days = dataDays;
     }
 
-    public ArrayList<Day> getDays() {
+    public static ArrayList<Day> getDays() {
         return days;
     }
 
     public static void setDays(ArrayList<Day> days) {
         WeekDataAccess.days = days;
     }
 
-    protected void save(){
+    /**
+     * Writes the contents of the days class variable onto a .ser file, persisting data
+     * @param days an ArrayList whose indices contain two hashmaps that contain events and tasks separately, representing the
```

Comment:
Technically the indices Day objects which contain hashmaps. Kindly correct this.